backend/FwLite/FwLiteProjectSync/DryRunMiniLcmApi.cs

```diff
@@ -39,11 +39,29 @@
         return api.GetPartsOfSpeech();
     }
 
+    public async Task<PartOfSpeech?> GetPartOfSpeech(Guid id)
+    {
+        // TODO: Should we add GetPartOfSpeech to the read API? Then we could just do this:
```

Comment:
might as well, we kinda need it anyway since our update methods return what was updated.

Reply:
Done in commit 00134570.